### ISSUE TITLE
“Incorrect user” error appearing for all accounts since this evening

### ISSUE THREAD

**Issue by Nnk2** [gruijter/com.gruijter.magister#28]
I did not change anything, but am getting an “Incorrect user” error for two different accounts since this evening... on my homey. Same credentials still work OK on the iOS magister app..

**Comment by gruijter** [comment]
Thx 4 reporting. I have the same. Magister is apparently blocking api use. I'm trying to get in touch with them to get to a solution.

**Comment by Nnk2** [comment]
Thx!

------------------------
Sent from iPhone
------------------------

**Comment by rhannink** [comment]
Hallo,

Same Problem here, since a few days, the app is not Able to login anymore

**Comment by gruijter** [comment]
I've sent Iddink (owner of magister) 2 emails, but they are not responding so far. I will keep trying.

**Comment by Nnk2** [comment]
Thank you!

------------------------
Sent from iPhone
------------------------

**Comment by Nnk2** [comment]
It seems that the homey magister app does occasionally connect though: I got one grade pushed (through my own homey workflow) last night 🤨, but now have the “invalid user” again, while (still) using the correct credentials ...

**Comment by gruijter** [comment]
Yes, I noticed the same. So the api auth key is sometimes correct. Maybe the magister api is not very secure, or maybe they open it up every once in a while.

I still have not gotten an answer from Iddink by the way.... I have no clue why not, since they promised to get get back to me via email.

**Comment by rhannink** [comment]
I saw on the iPad app that i needed to login again and then I got Some additional AVG questions. Maybe this has something to do with it?

**Comment by gruijter** [comment]
Yes, that has everything to do with it. But Iddink still didnt respond to my request to contact me about this.

**Comment by rhannink** [comment]
@gruijter, 
Have you Heard anything from Iddenk yet concerning the authentication problem?

I have read on the Internet that Magister is [sold from Iddink to Sanoma ](https://www.acm.nl/sites/default/files/documents/2019-09/besluit-sanoma-learning-iddink-holding.pdf)

Maybe this also has something to do with it. The document States that the API has to be available under fair use policy.

Gr. Remco

**Comment by Nnk2** [comment]
I have deleted the Homey magister app (and missed a few important notifications that I otherwise would have received through my homey workflows), since it still does not work 😢. I will keep following this thread though, hoping for better news ...

Keep up the good work @gruijter !
